src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -158,6 +161,8 @@ def run_mdtest_multiple_variants(self, mdtest_params):
                 self.mdtest_cmd.items.update(params[4])
             self.mdtest_cmd.depth.update(params[5])
             self.mdtest_cmd.flags.update(params[6])
+            if params[7]:
+                self.intercept_lib = True
```

Comment:
```suggestion
            if params[7]:
                self.mdtest_cmd.env['LD_PRELOAD'] = '/usr/lib64/libpil4dfs.so'
```
And then after `execute_mdtest` below:
```
del self.mdtest_cmd.env['LD_PRELOAD']
```

Reply:
Thank you very much! Fixed as you suggested.